pages/webview/content_page.py

```diff
@@ -22,15 +22,53 @@ def books_containing(self):
 
     class BooksContaining(Region):
         _root_locator = (By.CLASS_NAME, 'booksContaining')
+        _overview_locator = (By.CLASS_NAME, "title")
         _book_list_locator = (By.CSS_SELECTOR, 'div > ul > li')
 
+        @property
+        def is_displayed(self):
+            return self.is_element_displayed(*self._book_list_locator)
+
         @property
         def book_list(self):
             return [self.Book(self.page, el) for el in self.find_elements(*self._book_list_locator)]
 
+        @property
+        def description(self):
+            self.find_element(*self._root_locator)
+            return self.find_element(*self._overview_locator).text
+
+        @property
+        def date_list(self):
+            return [self.Book(self.page, el).revision_date.text.split(":")[1:]
+                    for el in self.find_elements(*self._book_list_locator)]
+
+        @property
+        def author_list(self):
+            return [(self.Book(self.page, el).author.text.split(': ')[1:])
+                    for el in self.find_elements(*self._book_list_locator)]
+
         class Book(Region):
             _title_locator = (By.CSS_SELECTOR, 'div')
+            _author_locator = (By.CSS_SELECTOR, "li > ul > li:nth-child(1) > div")
+            _revision_date_locator = (By.CSS_SELECTOR, "ul > li:nth-child(2) > div")
+            _go_to_book_locator = (By.CSS_SELECTOR,
+                                   "ul > li > ul > li:nth-child(3) > div > a")
 
             @property
             def title(self):
                 return self.find_element(*self._title_locator).text
+
+            @property
+            def author(self):
+                return self.find_element(*self._author_locator)
+
+            @property
+            def revision_date(self):
+                return self.find_element(*self._revision_date_locator)
+
+            @property
+            def click_go_to_book_link(self):
+                self.find_element(*self._go_to_book_locator).click()
```

Comment:
You may want to use the `self.offscreen_click` method here. It tends to be "safer".